travis/licenses_golden/licenses_third_party

```diff
@@ -20965,14 +10154,11 @@ a copy of the Unicode data files and any associated documentation
 without restriction, including without limitation the rights to use,
 copy, modify, merge, publish, distribute, and/or sell copies of
 the Data Files or Software, and to permit persons to whom the Data Files
-or Software are furnished to do so, provided that
+or Software are furnished to do so, provided that either
 (a) this copyright and permission notice appear with all copies
-of the Data Files or Software,
+of the Data Files or Software, or
 (b) this copyright and permission notice appear in associated
-documentation, and
-(c) there is clear notice in each modified Data File or in the Software
-as well as in the documentation associated with the Data File(s) or
-Software that the data or software has been modified.
+Documentation.
 
 THE DATA FILES AND SOFTWARE ARE PROVIDED "AS IS", WITHOUT WARRANTY OF
 ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
```

Comment:
This definitely seems wrong, the sentence cuts off half-way

Reply:
This was extracted directly from the ICU license file:
    https://chromium.googlesource.com/chromium/deps/icu/+/master/LICENSE